src/helpers/withThemeContext.js

```diff
@@ -0,0 +1,10 @@
+import { ThemeContext } from 'grommet'
+import React from 'react'
+
+export default function withThemeContext (WrappedComponent, theme) {
```

Comment:
Non-blocking comment: now that we have `@zooniverse/react-components` published, we have this HOC available in the exports there if you don't want to maintain a copy of it separately here.